desc/objectives/_geometry.py

```diff
@@ -567,6 +576,12 @@ def __init__(
         self._surface_grid = surface_grid
         self._plasma_grid = plasma_grid
         self._use_softmin = use_softmin
+        self._use_signed_distance = use_signed_distance
+        if use_softmin and use_signed_distance:
+            warnings.warn(
+                "signed distance cannot currently" " be used with use_softmin=True!",
```

Comment:
string syntax issue? Either make the whole this a single string, or I think you are missing a `+` between the two strings

Reply:
black moved things around, but two strings will be assumed to be concatenated even without a + in between so it is only an aesthetic issue